src/endpoints/transfers/transfer.service.ts

```diff
@@ -65,7 +62,7 @@ export class TransferService {
       transactions.push(transaction);
     }
 
-    await this.transactionService.processTransactions(transactions, pagination.size <= 100, assets);
+    await this.transactionService.processTransactions(transactions, pagination.size <= 100, pagination.size <= 50);
```

Comment:
the constants can be extracted

Reply:
refactored vigorously, extracted in constants and taken them into account